Cloud Application Manager/Deploying Anywhere/kubernetes.md

```diff
@@ -0,0 +1,165 @@
+
+*IAM* authentication of various cloud providers is not supported, the Kubernetes cluster need to have certificate or password based authentication configured.
+
+On saving the provider, CAM starts to synchronize with the Kubernetes cluster.
+
+##### Getting the certificate and key from a Google Kubernetes cluster
+
+```Shell
+gcloud config set container/use_client_certificate True
+gcloud container clusters list
+gcloud container clusters get-credentials cam-9436-denes-pal-1 --zone=us-central1-a
+```
+
+And enable legacy authentication to your Kubernetes cluster.
+
+#### On Synchronization
+
+When synchronizing with Kubernetes provider, CAM starts to discover the API on the cluster.
+
+* Checks for supported API groups, versions, preferred versions
+* Queries each API and each supported version for a list of supported resource types
+* Gets a list of available namespaces
+* Creates a Deployment Policy Box with *default* or the first available namespace
+* Downloads a list of all resources in order to offer them for registration
+
+#### Deployment Policies for Kubernetes
+
+You can create deployment policies by navigating to ***Boxes -> New -> Deployment Policy*** and selecting *Kubernetes*.
+
+Deployment policies of Kubernetes are bound to a **Namespace**, deployments can happen only into the selected namespace. You can set the namespace by editing the Policy on the *Code* tab of the Policy Box.
+
+### Create a Template Box
+
+Go to ***Boxes -> New -> Template*** and select *Kubernetes* as type. Fill the parameters as usual. Mind that if you define any *Requirements* you also need to list those in a Deployment Policy as *Claims* in order to be able to deploy the Template Box using that Policy.
+
+A Kubernetes Template Boxes can have multiple template files, currently only in YAML format, but CAM also supports **Jinja syntax with YAML**. You can change the order of template files by dragging and moving with the mouse, or long-pressing and dragging on a touch-screen device. At least one template file is required to deploy a Template Box.
+
+It is also valid to have multiple resources defined in a single template file, separated by a line of three dashes, like in the snippet below:
+
+```Yaml
+...
+            memory: 100Mi
+        ports:
+        - containerPort: 80
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: frontend
+...
+```
+
+If you want to deploy the same Template Box into the same namespace multiple times, you need to **include CAM Service ID** in the name of deployed resources, like:
+
+```Yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: frontend{{ service}}
+...
+```
+
+***New Template*** dialog can either:
+
+* Create a blank template by specifying its filename (please follow **.yaml* convention)
```

Comment:
```suggestion
* Create a blank template by specifying its filename (please follow **.yaml** convention)
```

Reply:
Should render as *.yaml in italic, that's the reason for the "missing" asterisk.